urwid/wimp.py

```diff
@@ -227,17 +227,17 @@ def set_state(self, state: bool | Literal['mixed'], do_callback: bool = True) ->
         do_callback -- False to suppress signal from this change
 
         >>> changes = []
-        >>> def callback_a(cb, state, user_data):
+        >>> def callback_a(user_data, cb, state):
```

Comment:
@wardi looks like incorrect resolve order in `Signals.emit`: `cb` is part of `*args` -> put after `user_args` and before deprecated `user_arg`.